src/store/modules/TsModsModule.ts

```diff
@@ -118,6 +118,12 @@ export const TsModsModule = {
             return getters.cachedMod(mod).isLatest;
         },
 
+        /*** Was the last successful mod list update more than an hour ago? */
+        isModListOutdated(_state) {
```

Comment:
The `_` prefix is there to indicate the positional argument is required although it's not used by the function. Some linters would comlain about it without the prefix, but that's not true in this repo. Anyways, since the argument is used, it shouldn't be prefixed.